CSharp/README.md

```diff
@@ -206,6 +207,71 @@ var result=2;
 var result = 2;
 ```
 
+## Ordering
+
+### Within a class, struct, or interface, elements must be ordered
+
+1. Constants
+1. Fields
+1. Constructors
+1. Delegates
+1. Events
+1. Properties
+1. Methods
+1. Finalizers (Destructors)
+
+### Declaration keywords must follow certain order
+
+1. Access modifiers
+1. `static`
+1. All other keywords
+
+```csharp
+// Bad
+string private _name;

```

Comment:
Is this even valid syntax? According to [this grammar document](https://blogs.msdn.microsoft.com/lucian/2010/04/19/vb-and-c-grammar-for-vs2010/), it would not be.

(I know it is old, but I couldn't find any newer version.)